### DISCUSSION TITLE
About the data transfer after updating the software

### DISCUSSION THREAD

**Discussion by Userwei0418** [TriliumNext/Trilium#6236] [Q&A] [answered]
### Describe feature

If you have previously used the old version and now want to update to the new version, but the original data is very large and 5g, how can you quickly transfer it? directly dragging the db file can not open the new version of the software. It is too troublesome to export and import with a note.

**Comment by zadam** [comment, marked as answer]
Hi, when the new version starts up, it should upgrade the database. You shouldn't need to export/import.
